docs/index.rst

```diff
@@ -3,6 +3,10 @@
 
       SPDX-License-Identifier: Apache-2.0
 
+.. image:: img/unidist-logo-simple-628x128.png
```

Comment:
Let's also set `html_favicon` and `html_logo` in conf.py.

https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_logo
https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_favicon